packages/contracts-bedrock/test/L2/L2StandardBridge.t.sol

```diff
@@ -130,13 +131,123 @@ contract L2StandardBridge_Test is Bridge_Initializer {
         assertEq(address(l2ToL1MessagePasser).balance, 100);
     }
 
+    /// @dev Tests receive bridges successfully with custom gas token.
+    function test_receive_customGasToken_succeeds() external {
```

Comment:
We can make this functionality revert as it only introduces bug risk

Reply:
If `test_receive_customGasToken_withValue_reverts` is a fuzz test then you can delete this entire test bc the fuzz with hit 0